src/CoreBundle/Resources/translations/admin.en.xliff

```diff
@@ -1088,9 +1088,9 @@
                 <target>Edit node</target>
                 <note>@deprecated since 6.3.8 - no longer used.</note>
             </trans-unit>
-            <trans-unit id="chameleon_system_core.cms_module_page_tree.legend_connected_to_protected_page">
-                <source>chameleon_system_core.cms_module_page_tree.legend_connected_to_protected_page</source>
-                <target>Navigation node with restricted access page</target>
+            <trans-unit id="chameleon_system_core.cms_module_page_tree.legend_connected_to_selected_page">
+                <source>chameleon_system_core.cms_module_page_tree.legend_connected_to_selected_page</source>
+                <target>Page has connected navigation node</target>
                 <note>@deprecated since 6.3.8 - no longer used.</note>
```

Comment:
The new / renamed entry has a depreaction notice? Is it now again used?

Reply:
I only sorted this deprecated entries together and git shows it new now.
I checked it, it's an old item that isn't used anymore.